disperser/cmd/dataapi/config.go

```diff
@@ -28,6 +28,7 @@ type Config struct {
 	PrometheusApiAddr            string
 	SubgraphApiBatchMetadataAddr string
 	SubgraphApiOperatorStateAddr string
+	SubgraphApiPaymentsAddr      string
```

Comment:
I'm assuming all of these addresses are the same? aka pointing to satsuma ip:port? Or are these full URLs? "socket address" is a bit confusing